docker-compose.yml

```diff
@@ -37,13 +37,13 @@ services:
       CLAMAV_PORT: 3310
     working_dir: /opt/app
     volumes:
-      - ./packages/common:/opt/app/node_modules/@pins/common:ro # Replace the module to avoid symlink errors
-      - ./packages/clamav-rest-server:/opt/app:ro
+      - ./packages/common:/opt/app/node_modules/@pins/common # Replace the module to avoid symlink errors
```

Comment:
When I tried I couldn't see a means of not removing readonly here.
I also don't fully understand exactly how volumes in docker work, so apologies if this doesn't make sense

However, is this a risk at all, as it's a volume that is shared between multiple containers?
If there's no solution should we perhaps look to not use shared volumes in the virus scanner?

Reply:
I honestly don't think this matters because docker compose is only used locally, when we build the docker image we don't use docker compose so from a security perspective I don't see the benefit.